### PR TITLE
Added: support for stylelint version 11

### LINKED ISSUE #368: support stylelint@11
:star: https://github.com/kristerkari/stylelint-scss/blob/master/package.json#L47

### PR DESCRIPTION
**Pull request by bjankord**
Resolves #368

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -41,10 +41,10 @@
     "request": "^2.88.0",
     "rimraf": "^3.0.0",
     "semver": "^6.3.0",
-    "stylelint": "^10.1.0"
+    "stylelint": ">=10.1.0"
```
Comment (bjankord) [inline package.json]: I went with this syntax but could see updating this to `^11.0.0`.

Comment (kristerkari) [reply package.json]: Not a big thing, but would be better to use `^11.0.0` for consistency.

**Comment by kristerkari** [comment]
Thanks! Test seems to be passing, so this looks good. I'll release a new version tomorrow.

File: package.json (hunk 1 above)
Comment (bjankord) [reply package.json]: Updated in this commit: https://github.com/kristerkari/stylelint-scss/pull/369/commits/830a85a659c59ac27969e8fab6b6bca8a13cac54

**Pull request merged**